arlas-core/src/main/java/io/arlas/server/utils/CheckParams.java

```diff
@@ -136,6 +136,17 @@ public static void checkFilter(Filter filter) throws ArlasException {
                 }
             }
         }
+        checkDateFormat(filter.dateformat);
+    }
+
+    public static void checkDateFormat(String dateFormat) throws ArlasException {
+        if (!StringUtil.isNullOrEmpty(dateFormat)) {
+            try {
+                DateTimeFormat.forPattern(dateFormat);
+            } catch (IllegalArgumentException e) {
+                throw new InvalidParameterException("Invalid date format '" + dateFormat + "'");
```

Comment:
Add getMessage from the exception so that we know what is wrong with that format.

Reply:
Done :+1: